go.mod

```diff
@@ -12,3 +12,5 @@ require (
 )
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
+
+go 1.13
```

Comment:
This kept getting re-added for me too, unsure why 🤔

Reply:
Reverted.